File: api/catalina-dto/src/main/java/com/ucacue/dto/model/TransaccionDTO.java
package com.ucacue.dto.model;

import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
public class TransaccionDTO {


    private Date fechaIngreso;
    private double cantidadIngreso;
    private int estadoPago;
    private int mes;
    private int dia;
    private  int anio;
    private UsersDTO users;
}

File: fronEnd/src/app/components/models/TransaccionDTO.ts
import {UsersDTO} from './UsersDTO'; 
export class TransaccionDTO {
        constructor(
            public fechaIngreso:string,
            public cantidadIngreso: number,
            public estadoPago:number,
            public mes:number,
            public dia: number,
            public anio : number,
            public usersDTO:UsersDTO
        ){
           


        }
}

